### ISSUE TITLE
Orphan tag in autocomplete.

### ISSUE THREAD

**Issue by Rsslone** [Sonarr/Sonarr#1373]
I found someone post about this on reddit a year ago but no answer or update on the status of the feature. In short I made a spelling mistake when adding a tag in sonarr but now it shows up as a autocomplete entry. How can I remove it?

**Comment by M0UL** [comment]
tags do not get deleted automatically. the only way to remove them is to edit the database.

**Comment by Rsslone** [comment]
Can you give me a short explanation on how I can do that? What file or tool I need will be fine. 

Thanks in advance.

**Comment by Rsslone** [comment]
I've figured it out myself. Here is what I did for anyone who is also looking for the answer.

Downloaded a database editor - https://github.com/sqlitebrowser/sqlitebrowser/releases
Opened the nzbdrone.db file located "C:\ProgramData\NzbDrone" for default installs.
Under "Tables" right-click "Tags" and select "Browse Table"
Press the number on left associated with the cell you would like to remove.
Press the "Delete Record" button on top right.
